### DISCUSSION TITLE
Disable / Enable "Entity search" and "Assist" in Dashboard

### DISCUSSION THREAD

**Discussion by leofig-rj** [home-assistant/frontend#23485] [Dashboard]
Aiming for Homeassistant installed in rental homes where we can create Dashboards in Kiosk Mode to limit access to users, we found a security flaw through the "Entity search" and "Assist" in the Dashboard.
One solution would be to create a flag to Enable and Disable these features in the Dashboard configuration.
This would allow for a more reliable application for situations where we do not want to make all resources available to certain users.

**Comment by ParkerFamilyDevelopment** [comment]
YES! I was able to get around this by adding the kiosk mode hide overflow option but its extremely annoying to have to do this on each and every dashboard,
